specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/artifact.json

```diff
@@ -0,0 +1,1693 @@
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of resource group in which the workspace is located.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "workspaceName",
+            "in": "path",
+            "description": "The name of the workspace.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "origin",
+            "in": "path",
+            "description": "The origin of the Artifact.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "container",
+            "in": "path",
+            "description": "The container name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "hardDelete",
+            "in": "query",
+            "description": "If set to true. The delete cannot be revert at later time.",
+            "required": false,
+            "type": "boolean",
+            "default": false
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Artifact metadata deleted successfully."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Artifact": {
+      "type": "object",
+      "properties": {
+        "artifactId": {
+          "type": "string"
+        },
+        "origin": {
+          "type": "string"
+        },
+        "container": {
+          "type": "string"
+        },
+        "path": {
+          "type": "string"
+        },
+        "etag": {
+          "type": "string"
+        },
+        "createdTime": {
+          "format": "date-time",
+          "type": "string"
+        }
+      }
+    },
+    "PaginatedArtifactList": {
+      "description": "A paginated list of items and an optional link to the next page of items.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of items in this page.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Artifact"
+          }
+        },
+        "continuationToken": {
+          "description": "The token used in retrieving the next page.  If null, there are no additional pages.",
+          "type": "string"
+        },
+        "nextLink": {
+          "description": "The link to the next page constructed using the continuationToken.  If null, there are no additional pages.",
+          "type": "string"
+        }
+      }
+    },
+    "Stream": {
+      "type": "object",
```

Comment:
missing description for model definition and model properties

Reply:
fixed